python/dgl/convert.py

```diff
@@ -1688,36 +1775,52 @@ def to_networkx(g, node_attrs=None, edge_attrs=None):
     >>> nx_g.edges(data=True)
     OutMultiEdgeDataView([(1, 1, {'id': 0, 'h1': tensor([1.]), 'h2': tensor([0., 0.])}),
                           (2, 3, {'id': 1, 'h1': tensor([1.]), 'h2': tensor([0., 0.])})])
+
+    With a heterogeneous graph:
+
+    >>> g = dgl.heterograph({
+        ...     ('user', 'follows', 'user'): (torch.tensor([0, 1]), torch.tensor([1, 2])),
+        ...     ('user', 'follows', 'topic'): (torch.tensor([1, 1]), torch.tensor([1, 2])),
+        ...     ('user', 'plays', 'game'): (torch.tensor([0, 3]), torch.tensor([3, 4]))
+        ... })
+        ... g.ndata['n'] = {
+        ...     'game': torch.zeros(5, 1),
+        ...     'user': torch.ones(4, 1)
+        ... }
+        ... g.edata['e'] = {
+        ...     ('user', 'follows', 'user'): torch.zeros(2, 1),
+        ...     'plays': torch.ones(2, 1)
+        ... }
+    >>> nx_g = to_networkx(g, node_attrs=['n'], edge_attrs=['e'])
+    >>> nx_g.nodes(data=True)
+    NodeDataView({0: {'label': 'game', 'n': tensor([0.])},
```

Comment:
What if there are features called `'label'`? For example, `g.ndata['label'] = ...`. It might be better to expose an argument for users to specify the node feature name to use for node types.

Reply:
True, it would be better to make it configurable. Perhaps it is still convenient to keep a default name, for example `type` (more common in triple-based approaches) rather than `label`?

Reply:
Yes. That sounds good to me.